apps/patient/src/components/Card.tsx

```diff
@@ -0,0 +1,10 @@
+import { VStack } from '@chakra-ui/react';
+import React from 'react';
+
+export const Card = (props: { children: React.ReactNode | React.ReactNode[] }) => {
```

Comment:
think there's a more chakra-y way of doing this but fine for now

Reply:
probably